lbz/request.py

```diff
@@ -52,11 +52,11 @@ def _decode_base64(encoded: Union[str, bytes]) -> bytes:
         return base64.b64decode(encoded)
 
     @property
-    def raw_body(self) -> Union[bytes, str]:
+    def raw_body(self) -> Union[bytes, str, dict]:
         if not self._raw_body and self._body is not None:
-            if self._is_base64_encoded:
+            if self._is_base64_encoded and isinstance(self._body, bytes):
```

Comment:
actually this is correct now, it was also correct before if payload would be an str it would not trigger as the if was on _is_bas64 flag which should only be in place with bytes.

Reply:
But the `self._body` is coming directly from the event so, in the case of e.g. API Gateway, it will be always string. I would say, we cannot rely here only on `_is_base64_encoded` flag.

Anyway, you have improved that, so it looks like the problem is solved 😉